src/components/technologies/genericTechnologies/stor.py

```diff
@@ -653,8 +654,12 @@ def init_ramping_down_rate_operation_in(const):
                             # -rampingRate <= input[t] - input[t-1]
                             return (
                                 -ramping_rate
-                                <= self.input[t, self.main_car]
-                                - self.input[t - 1, self.main_car]
+                                <= self.input[
```

Comment:
The if statement above (l 642) is different from the other technologies - correct?

Reply:
yes because storage has no performance type 1